index.html

```diff
@@ -0,0 +1,1280 @@
+                    </div>
+                    
+                </article> 
+
+
+
+            </div>
+
+        </div>
+
+        <div class="p-ctn slide-ctn container-fluid">
+
+            <div class="mag-title d-flex justify-content-center">
+                <span class="n-logo"></span>  featured Slideshows
+            </div>
+
+            <div class="row  ed-row">
+                <article class="art art-feat col-sm-4">
+                    <div class="art-img">
+                            <div class="slide-caption"> 8</div>
+                            <img src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.jpg?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d" alt="">
+                    </div>
+                    <div class="slide-cnt">
+                            <h4> Relive the 'Star Wars' Saga at These Filming Locations </h4>
+                    </div>
+                    
+                </article> 
+                <article class="art art-feat col-sm-4">
+                    <div class="art-img">
+                            <div class="slide-caption"> 14</div>
+                            <img src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.jpg?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d" alt="">
+                    </div>
+                    <div class="slide-cnt">
+                            <h4> Relive the 'Star Wars' Saga at These Filming Locations </h4>
+                    </div>
+                    
+                </article> 
+
+                <article class="art art-feat col-sm-4">
+                    <div class="art-img">
+                            <div class="slide-caption"> 6</div>
+                            <img src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.jpg?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d" alt="">
+                    </div>
+                    <div class="slide-cnt">
+                            <h4> Relive the 'Star Wars' Saga at These Filming Locations </h4>
+                    </div>
+                    
+                </article>             
+
+            </div>
+        </div>
+
+
+        <div class="p-ctn reg-ctn container-fluid">
+
+            <div class="reg-title">
+            U.S.
+            </div>
+
+            <div class="row ">
+                <article class="art art-feat col-md-4">
+                    <div class="art-img">
+                            <img src="https://d.newsweek.com/en/full/1577671/andrew-cuomo.jpg?w=397&h=265&f=751e6792dd7fa42bdc72604cfb3b5d50" alt="">
+                    </div>
+                    <div class="art-cnt">
+                            <h4> Trump Says He 'Wouldn't Mind Running Against' Andrew Cuomo </h4>
+                    </div>
```

Comment:
All the H4 in the bottom categories have a larger font-size when you are in the mobile version, please adjust the property.